docs/updating/from_3.3/update_from_3.3.md

```diff
@@ -235,6 +235,34 @@ Run the following scripts:
     psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-3.3.8-to-3.3.9.sql
     ```
 
+### v3.3.14
+
+#### Update VCL
+
+Update your Varnish VCL file to align with [`docs/varnish/vcl/varnish5.vcl`](https://github.com/ezsystems/ezplatform-http-cache/blob/2.3/docs/varnish/vcl/varnish5.vcl).
```

Comment:
Don't you want to say that the highlighted lines below have been added? Something along the lines of "make sure that the updated file contains the highlighted additions"?